src/Controller/PullRequestController.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use App\Repository\PullRequestRepository;
+
+class PullRequestController extends AbstractController
+{
+    #[Route('/pull/request', name: 'app_pull_request')]
```

Comment:
this road name is different from that of line 20.
/pull/request and /pullRequest

they should have the same structure.